server/middleware/currentUser.ts

```diff
@@ -12,30 +12,6 @@ export type User = {
 }
 
 export default ({ userService }: Services) => {
-  const getActiveCaseload = async (req, res) => {
-    const { activeCaseLoadId, username } = req.session.userDetails
```

Comment:
A user will always  have a activeCaseloadId either when their account is created or by DPS setting a default activeCaseloadId